client/my-sites/customer-home/main.jsx

```diff
@@ -203,164 +275,200 @@ class Home extends Component {
-						<h6 className="customer-home__card-subheader">
-							{ translate( 'Grow my audience and earn money' ) }
-						</h6>
-						<VerticalNav className="customer-home__card-links">
-							<VerticalNavItem
-								path={ `/marketing/connections/${ siteSlug }` }
-								onClick={ () => expandToolsAndTrack( 'earn', 'share_site' ) }
-							>
-								{ translate( 'Share my site' ) }
-							</VerticalNavItem>
-							<VerticalNavItem
-								path={ `/marketing/tools/${ siteSlug }` }
-								onClick={ () => expandToolsAndTrack( 'earn', 'grow_audience' ) }
-							>
-								{ translate( 'Grow my audience' ) }
-							</VerticalNavItem>
-							<VerticalNavItem
-								path={ `/earn/${ siteSlug }` }
-								onClick={ () => expandToolsAndTrack( 'earn', 'money' ) }
-							>
-								{ translate( 'Earn money' ) }
-							</VerticalNavItem>
-						</VerticalNav>
-					</Card>
+					{ siteIsUnlaunched && (
+						<Card className="customer-home__launch-button">
+							<CardHeading>{ translate( 'Site Privacy' ) }</CardHeading>
+							<h6 className="customer-home__card-subheader">
+								{ translate( 'Your site is private' ) }
+							</h6>
+							<p>
+								{ translate(
+									'Only you and those you invite can view your site. Launch your site to make it visible to the public.'
+								) }
+							</p>
+							<Button primary={ isPrimary } onClick={ this.onLaunchBannerClick }>
+								{ translate( 'Launch my site' ) }
+							</Button>
+						</Card>
+					) }
+					{ ! siteIsUnlaunched && (
+						<Card>
+							<CardHeading>{ translate( 'My Site' ) }</CardHeading>
+							<h6 className="customer-home__card-subheader">
+								{ translate( 'Make changes to your site or view its current state' ) }
+							</h6>
+							<div className="customer-home__card-col">
+								<div className="customer-home__card-col-left">
+									{ isStaticHomePage ? (
+										<Button
+											href={ editHomePageUrl }
+											primary={ isPrimary }
+											onClick={ () => trackAction( 'my_site', 'edit_homepage' ) }
+										>
+											{ translate( 'Edit Homepage' ) }
+										</Button>
+									) : (
+										<Button
+											href={ `/post/${ siteSlug }` }
+											primary={ isPrimary }
+											onClick={ () => {
+												trackAction( 'my_site', 'write_post' );
+											} }
+										>
+											{ translate( 'Write Blog Post' ) }
+										</Button>
+									) }
+								</div>
+								<div className="customer-home__card-col-right">
+									<Button href={ site.URL } onClick={ () => trackAction( 'my_site', 'view_site' ) }>
+										{ translate( 'View Site' ) }
+									</Button>
+								</div>
+							</div>
+						</Card>
+					) }
+					{ ! siteIsUnlaunched && (
+						<Card className="customer-home__grow-earn">
+							<CardHeading>{ translate( 'Grow & Earn' ) }</CardHeading>
+							<h6 className="customer-home__card-subheader">
+								{ translate( 'Grow your audience and earn money' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Grow my audience and earn money' )` *ES Score: 9*